dev/app-shell/elements/persistent-arc.html

```diff
@@ -16,16 +16,32 @@
         db: db.child('arcs')
       };
     }
+    _getAdditionalRecipe() {
```

Comment:
Is there a more specific name, maybe _getManifestToPersist() would describe this better?

Reply:
Renamed to ExternalManifest since that seems most representative of what it is. External to the manifests already built in to the existing app shell.

Existing fields in the schema don't state whether they're persisted or not, so deviating there didn't seem helpful. It does seem like documenting with a comment in ArcMetadata.schema would be useful but could do in a separate change, and also, that would be prone to becoming stale since code could change and folks might forget to update the schema doc comments.